standard/docs/en/schema/merging.md

```diff
@@ -51,106 +52,163 @@ If the record is complete with versioning information, then the versioning secti
 
 .. jsoninclude:: ../examples/merging/merge-award-2.json
    :jsonpointer: /releases
-   :expand: releases, award
+   :expand: releases, tag, awards
    :title: awardTwo
 
 ```
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/merging/merged.json
-   :jsonpointer: 
-   :expand: records, compiledRelease, tender, award, tag
+   :jsonpointer:
+   :expand: records, compiledRelease, tag, tender, awards
    :title: record
 
 ```
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/merging/versioned.json
-   :jsonpointer: 
-   :expand: records,versionedRelease, tender, award, tag
+   :jsonpointer:
+   :expand: records, versionedRelease, tag, tender, awards
    :title: versioned
 
 ```
 
 </div>
 
-## Merging rules
+## Merging specification
 
-Order all the releases that share an `ocid` by their release `date`. Starting from the oldest release (old), compare it with the next oldest release (new), and apply the following rules.
+### Discarded fields
 
-### Objects
+In the release schema, `"omitWhenMerged": true` is declared on fields that should be discarded during merging. These are presently: `id`, `date` and `tag`.
 
-In the compiled record, if an object can be set to `null` according to its `type`, it should be treated as a single value. Otherwise, merge the **new** object into the **old** object by:
+The compiled release discards both the fields and their values, whereas the versioned release discards the fields but *moves* their values, as described below.
 
-* Overwriting all keys that exist in both **old** and **new** with the values from **new**
-* Add any key-value pairs that exist in **new**, but not in **old**
-* Remove any key that have their value explicitly set to `null` in **new**
-* Retain any keys from **old** that are not mentioned in **new**
+If `omitWhenMerged` is set to `false`, ignore it.
 
-If the value of a key-value pair is a list of objects, the list merge rules below should be used; otherwise, a list should be treated as a single value.
+```eval_rst
+.. note::
 
-### Lists
+  The compiled release presently uses the same schema as the release schema, which means that the ``id``, ``date`` and ``tag`` fields are required in a compiled release. We invite discussion on whether to change these requirements in a separate compiled release schema in issue `#330 <https://github.com/open-contracting/standard/issues/330>`__, and on how to identify and date compiled and versioned releases in issue `#834 <https://github.com/open-contracting/standard/issues/834>`__.
 
-There are two merge patterns for lists of objects: identifier merge, and whole list merge. 
+  In the meantime, an intermediate solution is to set ``tag`` to ``["compiled"]``, ``date`` to the date of the most recent release, and ``id`` to ``{ocid}-{date}``, like in the `reference implementation <#reference-implementation>`__ of the merge routine.
```

Comment:
Suggest every time "merge routine" is mentioned, it is hyperlinked back to the relevant heading. This supports users who make heavy use of swapping in their short-term memories ;-)

Reply:
Done throughout the merge routine section in 50ee7f9